tools/integration_tests/rapid_appends/appends_test.go

```diff
@@ -201,6 +201,35 @@ func (t *RapidAppendsSuite) TestContentAppendedInNonAppendModeNotVisibleTillClos
 	assert.Equal(t.T(), expectedContent, contentAfterClose)
 }
 
+func (t *RapidAppendsSuite) TestAppendsToFinalizedObjectNotVisibleUntilClose() {
+	t.fileName = fileNamePrefix + setup.GenerateRandomString(5)
+	// Create Finalized Object in the GCS bucket.
+	client.CreateObjectInGCSTestDir(
+		ctx, storageClient, testDirName, t.fileName, initialContent, t.T())
+
+	// Append to the finalized object from the primary mount.
```

Comment:
add a comment here that the size of the data ensures that first block is flushed when streaming writes are enabled